image_explorer/image_explorer.py

```diff
@@ -134,14 +132,8 @@ def hotspot_coordinates_centered(self):
 
         return schema_version > 1
 
-    def author_view(self, context=None):
```

Comment:
Are we aware of the impact of the removal of this method?

Reply:
should i dont remove this?

Reply:
I don't know, I was just asking that. But I see that this is only used to run the applyHotspotButtonStyle function